user/urls.py

```diff
@@ -5,7 +5,7 @@
 urlpatterns = [
     url(r'^login/$', views.login, name='account_login'),
     url(r'^callback/(?P<provider>[0-9A-Za-z_\-]+)/$', views.callback, name='callback'),
-    url(r'^signup/$', views.signup, name='account_signup'),
+    url(r'^signup/(?P<u_type>[a-z_\-]{1,10})/$', views.signup, name='account_signup'),
```

Comment:
Add a redirect for `signup/` to `signup/hacker` for backwards compatibility pls.

Reply:
Done